stable/aws-pod-identity-webhook/values.yaml

```diff
@@ -0,0 +1,24 @@
+tls_secret_name: pod-identity-webhook
```

Comment:
Can you document these added fields in the REAME?

Reply:
Ah yes, I forgot to add these.